lib/themes/makeBraidTheme.ts

```diff
@@ -51,41 +50,40 @@ const normaliseSizingToCapHeight = (typography: BraidTokens['typography']) => {
 };
 
 const decorateTokens = (braidTokens: BraidTokens) => {
-  const { color, typography, ...restTokens } = braidTokens;
-
-  const getActiveColor = (x: string) =>
-    isLight(x) ? darken(0.1, x) : darken(0.05, x);
-
-  const getHoverColor = (x: string) =>
-    isLight(x) ? darken(0.05, x) : lighten(0.05, x);
-
-  const decoratedTokens = {
-    color: {
-      ...color,
-      background: {
-        ...color.background,
-        formAccentActive: getActiveColor(color.background.formAccent),
-        formAccentHover: getHoverColor(color.background.formAccent),
-        brandAccentActive: getActiveColor(color.background.brandAccent),
-        brandAccentHover: getHoverColor(color.background.brandAccent),
-        criticalActive: getActiveColor(color.background.critical),
-        criticalHover: getHoverColor(color.background.critical),
-        infoLight: getLightVariant(color.background.info),
-        promoteLight: getLightVariant(color.background.promote),
-        criticalLight: getLightVariant(color.background.critical),
-        positiveLight: getLightVariant(color.background.positive),
-        cautionLight: getLightVariant(color.background.caution),
-        neutralLight: getLightVariant(color.background.neutral),
-      },
-    },
-    typography: normaliseSizingToCapHeight(typography),
+  const {
+    border,
+    color,
+    contentWidth,
+    displayName,
+    grid,
+    name,
+    shadows,
+    space,
+    touchableSize,
+    transforms,
+    transitions,
+    typography,
+  } = braidTokens;
+
+  const treatTokens = {
+    border,
     breakpoint: breakpoints,
-    ...restTokens,
+    color,
+    contentWidth,
+    displayName,
+    grid,
+    name,
+    shadows,
+    space,
+    touchableSize,
+    transforms,
+    transitions,
+    typography: normaliseSizingToCapHeight(typography),
```

Comment:
FYI, I made this change so we don't have to worry about accidentally leaking new tokens into the treat theme.